### PR TITLE
OSS 2.6 release notes

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/docs-v2/issues/4665

Release notes for OSS 2.6

### CODE REVIEW

**Review by jeffreyssmith2nd — CHANGES_REQUESTED** [review]
Just throwing a blocker on here as I know there's a change coming this morning that I would like to add to the documentation

File: content/influxdb/v2.6/reference/release-notes/influxdb.md [hunk 1]
```diff
@@ -7,6 +7,26 @@ menu:
...
+ - Optimize [`SHOW FIELD KEY CARDINALITY`](/influxdb/v2.6/query-data/influxql/explore-schema/#show-field-key-cardinality) command.
+ - Restrict file permissions by default.
+ - Fix migration handling when already defined remotes exist.
+ - Fix default max-age transformation from seconds to one week.
+ - Fix error handling in user routes.
+
+### Maintenance
+
+- Upgrade to [Go 1.18.9](https://go.dev/doc/go1.18)
+- Upgrade to [Flux 0.191.0](/flux/v0.x/release-notes/#v01870-2022-10-17)
```
Comment (sanderson) [inline content/influxdb/v2.6/reference/release-notes/influxdb.md:29]:
```suggestion
## v2.6.0 [TBD]

### Features

- Add the following `influxd` utilities:
  - [`influxd inspect check-schema`](/influxdb/v2.6/reference/cli/influxd/inspect/check-schema/)
  - [`influxd inspect merge-schema`](/influxdb/v2.6/reference/cli/influxd/inspect/merge-schema/)
  - [`influxd inspect report-db`](/influxdb/v2.6/reference/cli/influxd/inspect/report-db/)
- Perform package validation during the build process.

### Bug Fixes

- Optimize [`SHOW FIELD KEY CARDINALITY`](/influxdb/v2.6/query-data/influxql/explore-schema/#show-field-key-cardinality)
  InfluxQL command.
- Correctly convert replication maximum age setting to seconds.
- Fix internal metadata migrations when a replication remote already exists.
- Update `/me` and `/users` routes to match documentation.
- By default, restrict file permissions to 0750 (`drwxr-x---`).
- Correctly handle `NaN` values returned from InfluxDB scrapers.

### Maintenance

- Upgrade to Go 1.18.9.
- Upgrade Flux to v0.191.0.
```

Comment (sanderson) [reply content/influxdb/v2.6/reference/release-notes/influxdb.md:29]: Had this on another branch.

File: content/influxdb/v2.6/reference/release-notes/influxdb.md [hunk 2]
```diff
@@ -7,6 +7,26 @@ menu:
...
+
+### Bug fixes
+
+ - Optimize [`SHOW FIELD KEY CARDINALITY`](/influxdb/v2.6/query-data/influxql/explore-schema/#show-field-key-cardinality) command.
+ - Restrict file permissions by default.
+ - Fix migration handling when already defined remotes exist.
+ - Fix default max-age transformation from seconds to one week.
+ - Fix error handling in user routes.
+
+### Maintenance
```
[the remark this answers is not in the document]
Comment (jeffreyssmith2nd) [reply content/influxdb/v2.6/reference/release-notes/influxdb.md:26]: This should encompass the last PR that I am merging today.

File: content/influxdb/v2.6/reference/release-notes/influxdb.md [hunk 3]
```diff
@@ -7,6 +7,30 @@ menu:
...
+
+- Optimize [`SHOW FIELD KEY CARDINALITY`](/influxdb/v2.6/query-data/influxql/explore-schema/#show-field-key-cardinality)
+  InfluxQL command.
+- Correctly convert replication maximum age setting to seconds.
+- Fix internal metadata migrations when a replication remote already exists.
+- Update `/me` and `/users` routes to match documentation.
+- By default, restrict file permissions to 0750 (`drwxr-x---`).
+- Correctly handle `NaN` values returned from InfluxDB scrapers.
+
+### Maintenance
```
[the remark this answers is not in the document]
Comment (jeffreyssmith2nd) [reply content/influxdb/v2.6/reference/release-notes/influxdb.md:30]: Reading the previous suggestion after changes

**Pull request merged**